pkg/operator/hive/hive_controller.go

```diff
@@ -223,6 +261,41 @@ func (r *ReconcileHiveConfig) Reconcile(request reconcile.Request) (reconcile.Re
 		Namespace: hiveNamespace,
 	})
 
+	if r.syncAggregatorCA {
+		// The reason we use the configmap lister here and not the regular client is that this controller is only watching
+		// resources in the 'hive' namespace. Trying to fetch from a different namespace results in a NotFound error.
+		aggregatorCAConfigMap, err := r.managedConfigCMLister.ConfigMaps(managedConfigNamespace).Get(aggregatorCAConfigMapName)
+		// If an error other than not found, retry. If not found, it means we don't need to do anything with
+		// admission pods yet.
+		switch {
+		case errors.IsNotFound(err):
+			hLog.Warningf("configmap %s/%s was not found, will not sync aggregator CA with admission pods", managedConfigNamespace, aggregatorCAConfigMapName)
+		case err != nil:
+			hLog.WithError(err).Errorf("cannot retrieve configmap %s/%s", managedConfigNamespace, aggregatorCAConfigMapName)
+			return reconcile.Result{}, err
+		case err == nil:
+			caHash := computeHash(aggregatorCAConfigMap.Data)
+			hLog.WithField("hash", caHash).Debugf("computed hash for configmap %s/%s", managedConfigNamespace, aggregatorCAConfigMapName)
+			if instance.Status.AggregatorClientCAHash != caHash {
+				hLog.WithField("oldHash", instance.Status.AggregatorClientCAHash).
+					Debugf("previous hash is not equal to new hash, redeploying admission")
```

Comment:
Debugf => Info, and maybe log "configmap %s/%s has changed, restarting hiveadmission pods".